src/auth/AxiosJwtAuthService.js

```diff
@@ -75,6 +75,30 @@ class AxiosJwtAuthService {
       });
   }
 
+  /**
+   *
+   * @param {HttpClient} client An axios instance.
+   * @param {Array|Function} middlewares The middleware to be applied.
```

Comment:
[nit/clarification] The JSDoc here says `middlewares` can be a function as well? I think the `Array|Function` bit comes from each element within the `middlewares` array, but the function arg passed to `applyMiddleware` would always be an array.